tests/rules/method.js

```diff
@@ -366,6 +366,12 @@ eslintTester.run("method", rule, {
         },
         {
             code: "(info.current = n.insertAdjacentHTML)('beforebegin', 'innocent')",
+        },
+        {
+
+            // #214: We also allow *harmful* parameters.
+            code: "let l = ['afterend', 'harmless']; foo.insertAdjacentHTML(...l);",
```

Comment:
Nit, would you mind to also add an additional test case for `foo.insertAdjacentHTML(something)`, that was also failing (the fix should cover both, but I think it may be worth adding to make sure that stays the case, e.g. when we may be applying more changes for #214).